### PR TITLE
feat: space panel removed and pinned items empty state added

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes: #4077 
### Description:

https://user-images.githubusercontent.com/67699259/228231786-1bffb232-c59c-4f05-861c-cbe640520193.mov

### CODE REVIEW

**Comment by owlas** [comment]
This looks great!

**Review by owlas — APPROVED** [review]
Code looks good, are there still mantine spacing changes to make? See // todo

**Comment by yasmin-e** [comment]
> 

nope, I added that one for future Mantine theming

**Review by PriPatel — APPROVED** [review]

File: packages/e2e/cypress/e2e/createProjects.cy.ts [hunk 1]
```diff
@@ -134,8 +134,8 @@ const testCompile = () => {
         .click();
     cy.url().should('include', '/home', { timeout: 30000 });
     cy.contains('Welcome, David');
-    cy.findByText('Shared');
-    cy.findByText('Spaces');
+    cy.findByText('No Pinned items');
```
Comment (IrakliJani) [inline packages/e2e/cypress/e2e/createProjects.cy.ts:137]: e2e is failing on this

File: packages/frontend/src/components/PinnedItemsPanel/index.tsx [hunk 2]
```diff
@@ -55,6 +60,40 @@ const PinnedItemsPanel: FC<Props> = ({ projectUuid, organizationUuid }) => {
                     : 'Your data team have pinned these items to help guide you towards the most relevant content!',
             }}
         />
+    ) : userCanUpdateProject && pinnedItems.length <= 0 ? (
+        // FIXME: update width with Mantine widths
+        <Card
+            w={900}
+            withBorder
+            style={{ backgroundColor: theme.colors.gray[1] }}
```
Comment (IrakliJani) [inline packages/frontend/src/components/PinnedItemsPanel/index.tsx:68]: it is better to use `sx` prop which has a theme awareness so you don't have to use `useMantineTheme`.

```tsx
sx={(theme) => ({
  backgroundcolor: theme.colors.gray[1],
})
```

File: packages/frontend/src/components/PinnedItemsPanel/index.tsx [hunk 3]
```diff
@@ -55,6 +60,40 @@ const PinnedItemsPanel: FC<Props> = ({ projectUuid, organizationUuid }) => {
                     : 'Your data team have pinned these items to help guide you towards the most relevant content!',
             }}
         />
+    ) : userCanUpdateProject && pinnedItems.length <= 0 ? (
+        // FIXME: update width with Mantine widths
+        <Card
+            w={900}
```
Comment (IrakliJani) [inline packages/frontend/src/components/PinnedItemsPanel/index.tsx:66]: do we need to use `w={900}` here? or can we just say `w="100%"`

**Review by IrakliJani — CHANGES_REQUESTED** [review]
minor changes needed

File: packages/frontend/src/components/PinnedItemsPanel/index.tsx (hunk 3 above)
Comment (yasmin-e) [reply packages/frontend/src/components/PinnedItemsPanel/index.tsx]: removed it completely, card inherits parent width 👍🏼

File: packages/frontend/src/components/PinnedItemsPanel/index.tsx (hunk 2 above)
Comment (yasmin-e) [reply packages/frontend/src/components/PinnedItemsPanel/index.tsx]: 👍🏼

**Review by IrakliJani — APPROVED** [review]
looks great!!! great job @yasmin-e @PriPatel

**Pull request merged**